charts/simple-app/values.yaml

```diff
@@ -110,17 +110,11 @@ startupProbe:
 # livenessProbe will be applied to the proxySidecar container instead if that
 # is enabled.
 livenessProbe:
-  httpGet:
-    path: /
-    port: http
 
 # -- A PodSpec container "readinessProbe" configuration object. Note that this
 # readinessProbe will be applied to the proxySidecar container instead if that
 # is enabled.
 readinessProbe:
-  httpGet:
```

Comment:
Same. Type conversion.

Reply:
⛔️ Blocking comment.

Reply:
going to leave as is and all probes can be updated in a followup PR following best typing practices.